src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,31 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+public class DoneCommand extends Command {
+    public static final String COMMAND_DONE = "done";
+    private final int taskNumber;
+
+    public DoneCommand(String arguments) {
```

Comment:
Maybe you can consider changing the name of the string variable to something like "argumentString" or "argumentStr".